troupon/payment/models.py

```diff
@@ -26,3 +27,18 @@ class TransactionHistory(models.Model):
                                        null=True,
                                        blank=True)
     user = models.ForeignKey(User)
+
+
+class IndividualTransactions(models.Model):
+    price = models.IntegerField()
+    advertiser = models.ForeignKey('deals.Advertiser')
+    title = models.CharField(max_length=100, null=False, blank=False)
+    description = models.TextField(blank=True, default='')
+    transaction_id = models.CharField(max_length=100,
+                                      null=False,
+                                      blank=False,
+                                      default='')
+    transaction_status = models.CharField(max_length=100,
```

Comment:
**Code Quality: Nitpick**

Put in choices for this field just to ensure we are not mistakenly inputting extraneous values.